.kokoro/continuous/doctest.cfg

```diff
@@ -0,0 +1,17 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Could you add the google3 one first? I want to see this job trigger before we merge this PR.

Reply:
For sure, I'm working through some presubmit issues on that one right now.